src/app/states/activeUserState.ts

```diff
@@ -0,0 +1,115 @@
+import {User} from '../interfaces/user';
+import {Action, Selector, State, StateContext} from '@ngxs/store';
+import {Injectable, NgZone} from '@angular/core';
+import {UserChangeLanguage, UserErrors, UserLogin, UserLogout, UserRegister} from './actions/activeUser.actions';
+import {AuthService} from '../services/authService/auth.service';
+import {Router} from '@angular/router';
+import {LanguageSetting} from '../configs/language-settings.config';
+
+export class ActiveUserStateModel {
```

Comment:
State models sollten immer interfaces sein:
```suggestion
export interface IActiveUserStateModel {
```